### ISSUE TITLE
Subscriptions &quot;shopping cart&quot; as input to new deletion request

### ISSUE THREAD

**Issue by ericvaandering** [dmwm/PHEDEX#635]
Original Savannah ticket 64381 reported by wildish on Tue Mar 16 04:21:39 2010.

Dear phedex developers, 

as you've probably noticed, analysis operations has gained a fair bit  
of operational experience within the last few months
with using phedex to host data in central space at T2s.

we are right now at the tail end of our first big &quot;deletion request&quot;  
campaign where we reduced the data hosted in central space by about 1PB.
This was a bit of a pain to do, and we would like to suggest two small  
improvements to the subscriptions interface.
1. Most importantly, we would like to have a column in the  
   subscriptions page that would allow us to click certain rows, and then
    generate deletion requests for all those rows that were clicked.  
   Basically, it is a painful exercise to manage  many tens of datasets  
   across
    a few tens of sites. The most powerful page in phedex is the  
   subscriptions page, with all of its &quot;Show Options&quot;. It would thus be  
   most natural
    for us to allow deletions from the subscriptions page.
2. It would be nice if the subscriptions page had a last row that  
   provides the totals. I.e. we would often like to make a selection using
    &quot;Show Options&quot; and not only get the individual items back, but  
   also the total for things like &quot;Node Bytes&quot;.

Both James and fkw are at CERN during cms week. So if you think a  
brief meeting to discuss this is useful, then let's plan for it.

Thanks, Frank
